.changeset/new-melons-tell.md

```diff
@@ -0,0 +1,6 @@
+---
+'@hyperlane-xyz/sdk': minor
```

Comment:
Feels more like a patch update than a minor one to me but nbd

Reply:
would agree if it was just the warp verification fix, but went with minor because of the ContractVerifier update to pickup apikeys